lib/content-services/src/lib/document-list/components/file-auto-download/file-auto-download.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { MatDialogRef } from '@angular/material/dialog';
+import { FileAutoDownloadActionsEnum } from '../../models/file-auto-download-actions.enum';
+
+@Component({
+  selector: 'adf-file-auto-download',
+  templateUrl: './file-auto-download.component.html',
+  styleUrls: ['./file-auto-download.component.scss']
+})
+export class FileAutoDownloadComponent {
+
+    constructor(private dialogRef: MatDialogRef<FileAutoDownloadComponent>) {}
+
+    onCancel() {
```

Comment:
as you are checking only for the "DOWNLOAD" result, you don't need this method when using "mat-dialog-close" attribute with the button